sql/core/src/main/scala/org/apache/spark/sql/streaming/ui/StreamingQueryStatisticsPage.scala

```diff
@@ -305,7 +305,7 @@ private[ui] class StreamingQueryStatisticsPage(parent: StreamingQueryTab)
         <tr>
           <td style="vertical-align: middle;">
             <div style="width: 160px;">
-              <div><strong>Aggregated Number Of Evicted State Rows{SparkUIUtils.tooltip("Aggregated number of state rows evicted from the state.", "right")}</strong></div>
+              <div><strong>Aggregated Number Of Removed State Rows{SparkUIUtils.tooltip("Aggregated number of state rows evicted from the state. Normally it means the number of rows evicted from the state because of passing watermark. Except in flatMapGroupWithState users can manually remove the state", "right")}</strong></div>
```

Comment:
last nit: 
Aggregated number of state rows evicted from the state. Normally it means the number of rows evicted from the state because of passing watermark. Except in flatMapGroupWithState users can manually remove the state

to

Aggregated number of state rows `removed` from the state. Normally it means the number of rows evicted from the state `because watermark has passed, e`xcept in flatMapGroupWithState users can manually remove the state`.`